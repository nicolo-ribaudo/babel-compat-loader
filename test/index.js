const path = require("path");
const loadFixtures = require("./helpers/fixtures-loader.js");
const runFixture = require("./helpers/fixtures-runner.js");

const loaderPath = path.resolve(__dirname, "../src/index.mjs");

const fixtures = loadFixtures(path.resolve(__dirname, "./fixtures"));

fixtures.forEach(fixtures => runFixture(fixtures, loaderPath));
