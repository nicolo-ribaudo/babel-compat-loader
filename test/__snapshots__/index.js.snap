// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`already-required-executed-once - babel 6: stderr 1`] = `undefined`;

exports[`already-required-executed-once - babel 6: stdout 1`] = `"sub-module executed"`;

exports[`already-required-executed-once - babel 7: stderr 1`] = `undefined`;

exports[`already-required-executed-once - babel 7: stdout 1`] = `"sub-module executed"`;
